inactiveIssues/index.ts

```diff
@@ -0,0 +1,202 @@
+import { addDays, distinct } from '../shared/utils';
+import { completeFunction } from '../shared/functions';
+import { IssueNode } from '../shared/models';
+import { getAllMilestones, getAllGitHubIssuesRecursively, commentGitHubIssue, closeGitHubIssue } from '../shared/github';
+
+module.exports = (context) => {
+    const githubApiHeaders = {
+        'User-Agent': 'github-bot-uwp-toolkit',
+        'Authorization': 'token ' + process.env.GITHUB_BOT_UWP_TOOLKIT_ACCESS_TOKEN
+    };
+
+    getAllMilestones(
+        githubApiHeaders,
+        process.env.GITHUB_BOT_UWP_TOOLKIT_REPO_OWNER,
+        process.env.GITHUB_BOT_UWP_TOOLKIT_REPO_NAME,
+        (milestones) => {
+            const currentMilestone = milestones
+                .filter(m => m.state === 'OPEN' && !!m.dueOn)
+                .sort((m1, m2) => new Date(m1.dueOn).getTime() - new Date(m2.dueOn).getTime())
+            [0];
+
+            getAllGitHubIssuesRecursively(
+                githubApiHeaders,
+                process.env.GITHUB_BOT_UWP_TOOLKIT_REPO_OWNER,
+                process.env.GITHUB_BOT_UWP_TOOLKIT_REPO_NAME,
+                null,
+                (issues) => {
+                    const exclusiveLabels = [
+                        'PR in progress',
+                        'work in progress',
+                        'help wanted',
+                        'uservoice-entry-created',
+                        'mute-bot'
+                    ];
+
+                    // only check issues in the current milestone or not in a milestone
+                    // only check issues without exlusive labels
+                    const issuesToCheck = issues
+                        .filter(issue => {
+                            return (!issue.milestone || issue.milestone.number <= currentMilestone.number);
+                        })
+                        .filter(issue => {
+                            return !isIssueContainsExclusiveLabels(issue, exclusiveLabels);
+                        });
+
+                    const issuesInTheCurrentMilestone = issuesToCheck
+                        .filter(issue => issue.milestone && issue.milestone.number === currentMilestone.number);
+
+                    const issuesNotInMilestone = issuesToCheck
+                        .filter(issue => issue.milestone && issue.milestone.number < currentMilestone.number);
```

Comment:
I might have missed it, it doesn't seem to check for issues that don't have milestone.

Reply:
Oh, good catch. Should be `!issue.milestone || issue.milestone.number < currentMilestone.number`